pkg/config/global_test.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"os"
+	"testing"
+
+	"github.com/helmfile/helmfile/pkg/envvar"
+	"github.com/stretchr/testify/require"
```

Comment:
update it like below:

"github.com/stretchr/testify/require"

"github.com/helmfile/helmfile/pkg/envvar"